extensions/wikia/AssetsManager/config.php

```diff
@@ -2613,3 +2613,29 @@
 		'//extensions/wikia/JWPlayerTag/styles/jwplayertag.scss'
 	]
 ];
+
+$config['recommended_video_css'] = [
+	'type' => AssetsManager::TYPE_CSS,
```

Comment:
why do we need both, css and scss packages?

Reply:
That's because we're loading one css and one scss file.

Reply:
maybe we could just import CSS in the scss?